grievance_social_protection/apps.py

```
@@ -91,34 +126,236 @@ def get_grievance_type_options_msg(types):
-                    key,
-                    resolution_times
-                )
-                dict_field[key] = resolution_times
-            else:
-                if ',' not in value:
-                    logger.warning("Invalid input. Configuration should contain two integers "
-                                   "representing days and hours, separated by a comma.")
+        if dict_field:
+            for key in dict_field:
+                value = dict_field[key]
+                if value in ['', None]:
+                    resolution_times = cfg.get("resolution_times", DEFAULT_TIME_RESOLUTION)
+                    logger.warning(
+                        '"%s" has no value for resolution. The default one is taken as "%s".',
+                        key,
+                        resolution_times
+                    )
+                    dict_field[key] = resolution_times
                 else:
-                    parts = value.split(',')
-                    # Parse days and hours
-                    days = int(parts[0])
-                    hours = int(parts[1])
-                    # Validate days and hours
-                    if 0 <= days < 99 and 0 <= hours < 24:
-                        logger.info(f"Days: {days}, Hours: {hours}")
-                    else:
-                        logger.warning("Invalid input. Days must be between 0 and 99, "
-                                       "and hours must be between 0 and 24.")
+                    cls.__validate_resolution_time_format(value, f"default_resolution[{key}]")
+        
+        # Then, validate resolution_times in processed categories
+        processed_categories = cfg.get("processed_categories", {})
+        for category_name, category_info in processed_categories.items():
+            resolution_time = category_info.get('resolution_times')
+            if resolution_time:
+                cls.__validate_resolution_time_format(resolution_time, f"category '{category_name}'")
```

The returned boolean value from `__validate_resolution_time_format` isn't used her nor above. I see the validation is only called when resolution_time is explicitly configured. In the same vein as some of the previous comments on exception handling, I'm leaning towards throwing an error and fail dramatically if there's an invalid configuration instead of just logging the issue here, because the goal is to flag to the user when there's an invalid configuration and have the user correct it as soon as possible.